index.html

```diff
@@ -148,6 +148,30 @@ <h2>
         method.
         </li>
       </ul>
+      <p>
+        A payment method defines:
+      </p>
+      <dl>
+        <dt>
+          <dfn>Payment Method additional data type</dfn>
```

Comment:
I think this should be defined as: 

```suggestion
          <dfn data-dfn-for="Payment Method">additional data type</dfn>
```

Reply:
Done.